packages/acceptit-Tests.package/ACUserStoryFactory.class/class/userStoryTitle2.st

```diff
@@ -1,4 +1,4 @@
-as yet unclassified
+userStoryConstants
```

Comment:
rename this to `userstory constants` instead of cammel case please. Same with category constant